sites/public/cypress/support/e2e.js

```diff
@@ -12,7 +12,7 @@
 // You can read more here:
 // https://on.cypress.io/configuration
 // ***********************************************************
-import "@cypress/code-coverage/support"
+// import "@cypress/code-coverage/support"
```

Comment:
Should we remove this?

Reply:
I was able to add it back in by upgrading the version of it. I had originally commented it out because all of the cypress tests were failing after they had finished and I traced it back to this being the culprit. However, I forgot to go back and investigate since it wasn't happening in the partner tests.